workspaces.tf

```
@@ -0,0 +1,60 @@
+locals {
+  tfe_organization   = "mindtastic"
+  tfe_oauth_token_id = "ot-wwv4yMLk5usRLHzQ" # https://app.terraform.io/app/mindtastic/settings/version-control
```

```suggestion
  tfe_oauth_token_id = var.tfc_token # https://app.terraform.io/app/mindtastic/settings/version-control
```

If a variable is applicable to locals. Otherwise we should reference the terraform variable in the code below directly.